Providers/Scripts/2.4x-2.5x/Scripts/nxOMSAutomationWorker.py

```diff
@@ -175,6 +201,18 @@ def Get_Marshall(WorkspaceId, Enabled, AzureDnsAgentSvcZone):
 KEYRING_PATH="/etc/opt/omi/conf/omsconfig/keyring.gpg"
 LOCAL_LOG_LOCATION = "/var/opt/microsoft/omsagent/log/nxOMSAutomationWorker.log"
 
+def is_primary_workspace(mof_workspace_id):
+    """
+    Detect if the workspsace is secondary multi-homing workspace
+    Mulit-homing for OMS is 2 tiered, one primary and multiple secondary workspaces are allowed
```

Comment:
Can you add a quick comment explaining how you know if it's a primary / secondary workspace. Seems like your logic is simply to look at the existing worker configuration and assuming that if it is already registered then it is primary. Is that enough??